src/IngredientEntity.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace GRUB; 
+
+/* Class for IngredientEntities */
+class IngredientEntity {
+    private $name;
+    
+    /**
+     * Constructor for IngredientEntity
+     *
+     * @param string $name Name for ingredient object
+     */
+    public function __construct(string $name){
+
+        $this->name = $name;
+    }
```

Comment:
This is not needed in this case, the PDO FETCH_CLASS mode will automatically instantiate the objects and populate the properties based on MySQL table names.

Reply:
When using FETCH_CLASS the only time you need a constructor is if you need to override parameters.